File: 0001 Code Chef/00014 Devouring Donuts/Implementation/0003 c#/DevouringDonuts.cs
using System;

class DevouringDonuts{
    static void Main(string[] args){
        string stringInput = Console.ReadLine();
        string[] stringArray = stringInput.Split(" ");
        int x = Convert.ToInt32(stringArray[0]);
        int y = Convert.ToInt32(stringArray[1]);
        Console.WriteLine(x*y);
    }
}

File: 0001 Code Chef/00014 Devouring Donuts/Implementation/0005 kotlin/DevouringDonuts.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var stringArray:Array<String> = stringInput.split(" ").toTypedArray();
        var x:Int? = stringArray[0].toIntOrNull();
        var y:Int? = stringArray[1].toIntOrNull();
        if(x != null && y != null){
            println(x*y);
        }
        }
    }